Classes/git/PBGitIndex.m

```diff
@@ -44,6 +44,7 @@ - (NSArray *)linesFromData:(NSData *)data;
 @interface PBGitIndex () {
 	dispatch_queue_t _indexRefreshQueue;
 	dispatch_group_t _indexRefreshGroup;
+	BOOL amend;
```

Comment:
Keep the property, but name the ivar `_amend`. That'll be less surprising (properties by default use a underscored ivar if there's one).